Dockerfile

```diff
@@ -28,6 +28,17 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
+ONBUILD ARG ODOO_MAX_CRON_THREADS=-1
+ONBUILD ARG ODOO_WORKERS=0
+ONBUILD ARG ODOO_LIMIT_TIME_REAL=120
+ONBUILD ARG ODOO_LIMIT_TIME_CPU=60
```

Comment:
In our experience it's quite better to increase these 2 defaults. Simple standard operations such as downloading a db backup never get done with these values. At least 10x please.